### DISCUSSION TITLE
您好大佬，为什么有的界面里面增加了html内容就会显示成空白页面了？

### DISCUSSION THREAD

**Discussion by oxn** [Tokinx/Wing#9] [讨论]
<style>
.cyan {
    background-color: #34A853; /* 绿色 */
    border: none;
    color: white;
    padding: 8px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 4px;
    font-family: "lucida grande", "lucida sans unicode", lucida, helvetica, "PingFang SC", "Hiragino Sans GB", "STHeiti Light", "Microsoft YaHei", "WenQuanYi Micro Hei";
}
.cyan:hover {
    box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24),0 17px 50px 0 rgba(0,0,0,0.19);
}
.cyan2 {background-color: #4285f4;} /* 蓝色 */
.cyan3 {background-color: #ea4335;} /* 红色 */
.cyan4 {background-color: #757575; color: black;} /* 灰色 */
.cyan5 {background-color: #555555;} /* 黑色 */
.cyan6 {background-color: #fbbc05;} /* 黄色 */

div.cyc {
	margin: 0px 0px 5px; padding: 5px; border: 1px solid rgb(212, 212, 212); border-image: none; width: 99%; color: rgb(0, 0, 0); background-image: linear-gradient(rgb(255, 255, 255), rgb(229, 238, 204) 100px); background-color: rgb(229, 238, 204);
}
textarea{

resize:none;

}
.cyk {
	padding: 6px 12px; border-radius: 4px; border: 1px solid rgb(204, 204, 204); transition:border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out, -webkit-box-shadow 0.15s ease-in-out; border-image: none; width: 99%; color: rgb(85, 85, 85); font-size: 14px; display: block; box-shadow: inset 0px 1px 1px rgba(0,0,0,0.075); background-image: none; background-color: rgb(255, 255, 255); -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075); -webkit-transition: border-color ease-in-out .15s, -webkit-box-shadow ease-in-out .15s; -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
}
.cyk:focus {
	border-color: rgb(102, 175, 233); outline: 0px; box-shadow: inset 0px 1px 1px rgba(0,0,0,0.075), 0px 0px 8px rgba(102,175,233,0.6); -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075), 0 0 8px rgba(102, 175, 233, .6);
}
.cytxt {
display: font-size: 16px; color: #303030; line-height: 1.2; font-family: "lucida grande", "lucida sans unicode", lucida, helvetica, "PingFang SC", "Hiragino Sans GB", "STHeiti Light", "Microsoft YaHei", "WenQuanYi Micro Hei"; word-wrap: break-word; word-break: break-all; ove flow: hidden;
}
.cytxtbak {
display: inline-block; font-size: 16px; color: #303030; line-height: 1.2; font-family: "lucida grande", "lucida sans unicode", lucida, helvetica, "PingFang SC", "Hiragino Sans GB", "STHeiti Light", "Microsoft YaHei", "WenQuanYi Micro Hei"; word-wrap: break-word; word-break: break-all; ove flow: hidden;
}
</style>
<DIV class="cyc">
<div class="cytxt" style="font-size:20px; text-align: center; font-weight:bold">JS转义工具-怸歪小站</DIV>
<P><DIV class="cytxt">将Js代码编码一下，转成非特殊字符，从而避免被wp自动转义</DIV></P>
<DIV align="center"><textarea class="cyk" id="jscode" onclick="focus();select()"; placeholder="在此输入内容" style="height:250px;"></textarea></DIV>
<DIV align="center"><input class="cyan" onclick="encodeJS();" value="编码" type="button" />&nbsp;&nbsp;&nbsp;<input class="cyan cyan2" onclick="decodeJS();" value="解码" type="button" />&nbsp;&nbsp;&nbsp;<input class="cyan cyan3" type="button" value="执行" onclick="testJS();" />&nbsp;&nbsp;&nbsp;<input class="cyan cyan6" type="button" value="清空" onclick="Empty();" />&nbsp;&nbsp;&nbsp;<input class="cyan cyan5" type="button" value="复制" onclick="copyinput()"/>
</DIV></DIV>
<DIV class="cytxt">★将上面编码过的内容，拷贝到你的 wordpress 正文中，即可,本程序转自 水很凉 的博客,CSS样式采用runoob.com和txttool.com的，如果喜欢可以拿过去直接用。</DIV>
<script type="text/javascript">
 function D(obj)
 {
     return document.getElementById(obj);
 }
 function DF(obj)
 {
     return document.getElementById(obj).value;
 }
 function encodeJS()
 {
    var es = escape(DF('jscode'));
    D('jscode').value = "<script type=\"text/javascript\">document.write(unescape('" + es +"'));<\/script>";
 }
 
 function decodeJS()
 {
    var regex = /unescape\('(.+?)'\)/i;
    D('jscode').value = unescape(regex.exec(DF('jscode'))[1]);
 }
</script>
 <SCRIPT LANGUAGE="JavaScript"> 
 // 执行	
 function testJS()
 {
    var win = window.open();
    win.document.write(DF('jscode'));
 }
// 清空
function Empty() {
        document.getElementById('jscode').value = '';
        document.getElementById('jscode').select();
    }
 // 复制
 function copyinput() 
 { 
 var input=document.getElementById("jscode");//input的ID值 
 input.select(); //选择对象 
 document.execCommand("Copy"); //执行浏览器复制命令 
 } 
</script>

**Comment by Tokinx** [comment]
`onclick="focus();select();"`

这个地方的分号你写错位置了，要在引号里面

**Comment by oxn** [comment]
收到。谢谢大佬

**Reply by oxn** [reply]
大佬样式失效了是因为没有加载个别的js文件吗？

**Reply by oxn** [reply]
http://119.8.30.111/wordpress-js-zhuanyi/

**Reply by Tokinx** [reply]
看了下是你html的累和css里的样式累没对应，你自己看下怎么改吧

**Reply by oxn** [reply]
谢谢大佬。我研究一下

**Comment by oxn** [comment]
![a](https://user-images.githubusercontent.com/3295116/169455018-51ff3175-aa9a-4c95-a833-dc9faeb9dfee.png)
![w](https://user-images.githubusercontent.com/3295116/169455028-471c5e47-9a46-4dcc-b965-64ee4cc7ca1e.png)
大佬我看了一下，新主题在审查时发现css加载不上，不知道什么原因，那个类我上网查一查学习学习，真的是不懂，小白一枚。谢谢大佬的细心答复，我接着去查找原因去。

**Reply by Tokinx** [reply]
可能是新主题的Vue把style标签给过滤掉了，你把css放到主题-自定义-额外的CSS里吧，暂时没有比较好的办法。
或者你使用子主题的开发模式，新建一个页面模板。

**Reply by oxn** [reply]
好的大佬，我试一下。我看外挂的css是可以的。内置的不行。

**Comment by oxn** [comment]
经过几天的琢磨，终于把那几个页面不显示的都给搞了，谢谢大佬的耐心知道，继续摸索中。
